docs/core/metrics.md

```diff
@@ -111,11 +111,12 @@ Visit the AWS documentation for a complete explanation for [Amazon CloudWatch co
 
 Metrics has three global settings that will be used across all metrics emitted. Use your application or main service as the metric namespace to easily group all metrics:
 
-| Setting                        | Description                                                                     | Environment variable           | Decorator parameter |
-| ------------------------------ | ------------------------------------------------------------------------------- | ------------------------------ | ------------------- |
-| **Metric namespace**           | Logical container where all metrics will be placed e.g. `ServerlessAirline`     | `POWERTOOLS_METRICS_NAMESPACE` | `namespace`         |
-| **Service**                    | Optionally, sets **service** metric dimension across all metrics e.g. `payment` | `POWERTOOLS_SERVICE_NAME`      | `service`           |
-| **Disable Powertools Metrics** | Optionally, disables all Powertools metrics                                     | `POWERTOOLS_METRICS_DISABLED`  | N/A                 |
+| Setting                        | Description                                                                     | Environment variable               | Decorator parameter |
+| ------------------------------ | ------------------------------------------------------------------------------- | ---------------------------------- | ------------------- |
+| **Metric namespace**           | Logical container where all metrics will be placed e.g. `ServerlessAirline`     | `POWERTOOLS_METRICS_NAMESPACE`     | `namespace`         |
+| **Service**                    | Optionally, sets **service** metric dimension across all metrics e.g. `payment` | `POWERTOOLS_SERVICE_NAME`          | `service`           |
+| **Function name**              | Function name used as dimension for the cold start metric                       | `POWERTOOLS_METRICS_FUNCTION_NAME` | `functionName`      |
+| **Disable Powertools Metrics** | Optionally, disables all Powertools metrics                                     | `POWERTOOLS_METRICS_DISABLED`      | N/A                 |
```

Comment:
Suggested `Optionally, disables metrics flushing`.

As a side note, I'd suggest to do a find & replace and make sure we're not using `Powertools` alone - should be at least `Powertools for AWS`, when in doubt, just refer to the name of the utility/feature if shorter.

Reply:
https://github.com/aws-powertools/powertools-lambda-java/pull/1863/commits/0017e8b10ffcf38b1741cef1a115eb9dfd17bd10